src/suresh/Shop.java

```diff
@@ -0,0 +1,65 @@
+package suresh;
+
+public class Shop {
+	
+	String customername;
+	//Integer items;
+	Integer Maggie = 50; 
+	Integer Dosa = 43;
+	Integer pouches =39;
+	Integer panipuri = 43;
+	Integer Masalas =73;
+	
+	void customer(String customername){
+		this.customername = customername;
+	}
+	
+	void quantity(Integer Maggie,Integer Dosa,Integer pouches,Integer panipuri,Integer Masalas){
+		
+		this.Maggie = Maggie;
+		this.Dosa = Dosa;
+		this.pouches = pouches;
+		this.panipuri = panipuri;
+		this.Masalas = Masalas;
+		System.out.println("<--------> Order Details <-------->");
+		System.out.println("Ordered Quantity- Maggie: " + Maggie);
+		System.out.println("Ordered Quantity- Dosa: " + Dosa);
+		System.out.println("Ordered Quantity- pouches: " + pouches);
+		System.out.println("Ordered Quantity- panipuri: " + panipuri);
+		System.out.println("Ordered Quantity- Masalas: " + Masalas);
+	}
+	void OutOfStock(){
+		if(Maggie>50) {
```

Comment:
this is wrong approach @sureshchukka never hard code values
please refer this code

https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/58